index/index.go

```diff
@@ -868,9 +868,9 @@ func (r *Reader) LabelValues(names ...string) (StringTuples, error) {
 		return nil, errors.Wrap(d.err(), "read label value index")
 	}
 	st := &serializedStringTuples{
-		l:      nc,
-		b:      d.get(),
-		lookup: r.lookupSymbol,
+		length:  nc,
```

Comment:
I need to read the code deeper, but I think this represents a the 32 bit big Indian version and not a length.

Reply:
I thought as according to [here](https://github.com/prometheus/tsdb/blob/3df5bcf39e9fb4dea3f70f9ec2bcb77c23271268/index/index.go#L937). What should I change it too?